server/src/main/java/io/druid/server/initialization/AuthenticatorMapperModule.java

```diff
@@ -125,7 +125,43 @@ public AuthenticatorMapper get()
         authenticatorMap.put(authenticatorName, authenticator);
       }
 
-      return new AuthenticatorMapper(authenticatorMap, authConfig.getEscalatedAuthenticator());
+      return new AuthenticatorMapper(authenticatorMap);
+    }
+  }
+
+  private static void validateProperties(Properties properties, List<String> authenticators)
+  {
+    String escalatorType = properties.getProperty("druid.escalator.type");
+    if (escalatorType == null) {
+      escalatorType = "allowAll";
```

Comment:
This should be a constant (static final somewhere) and ideally all instances of the string "allowAll" would reference that.

Reply:
Made "allowAll" a constant under AuthConfig